src/invidious/comments.cr

```diff
@@ -73,9 +73,9 @@ def fetch_youtube_comments(id, cursor, format, locale, thin_mode, region, sort_b
   contents = nil
 
   if response["onResponseReceivedEndpoints"]?
-    onResponseReceivedEndpoints = response["onResponseReceivedEndpoints"]
+    on_response_received_endpoints = response["onResponseReceivedEndpoints"]
```

Comment:
```suggestion
```

Reply:
There's a slight performance hit when you directly access the hash multiple times instead of saving to a variable first.

Reply:
Whaaat O.o Shouldn't that be optimized by the compiler?

Reply:
Apparently not. Here's the benchmark I did eariler:

```
Access via variable  58.21M ( 17.18ns) (± 1.30%)  0.0B/op        fastest
      Direct access  31.18M ( 32.07ns) (± 1.22%)  0.0B/op   1.87× slower
```

<details>
<summary>
Source code
</summary>

```crystal
require "benchmark"

hash = {
  "hello" => 10,
  "yeet"  => 20,
  "word"  => 30,
  "omg"   => 40,
  "test"   => {
    "a" => 1,
    "b" => 2,
    "3" => 4,
  },
}

Benchmark.ips do |test|
  test.report("Access via variable") {
    if inner = hash["test"]?
      inner.as(Hash(String, Int32)).each do |k, v|
        {k, v}
      end
    end
  }

  test.report("Direct access") {
    if hash["test"]?
      hash["test"].as(Hash(String, Int32)).each do |k, v|
        {k, v}
      end
    end
  }
end
```

</details>

I mean the performance hit is negligible, but it is still present.

Reply:
I added a 3rd option to that benchmark, to be similar to whet we hav in the code, and it seems that this is due to the variable assignment *inside the if statement*:

```crystal
  test.report("Access via variable (2)") {
    if hash["test"]?
      inner = hash["test"]
      inner.as(Hash(String, Int32)).each do |k, v|
        {k, v}
      end
    end
  }
```

```
    Access via variable  19.55M ( 51.16ns) (± 7.05%)  0.0B/op        fastest
Access via variable (2)  10.41M ( 96.03ns) (± 7.09%)  0.0B/op   1.88× slower
          Direct access  10.33M ( 96.82ns) (± 4.87%)  0.0B/op   1.89× slower
```

Reply:
Ah right! I guess I applied an extra layer of optimization there with the if statement... Either way, that method is faster and I've applied it to this PR with commit 36ebdb3.